src/components/SharedComponents/Button/common.ts

```diff
@@ -19,7 +21,8 @@ export type ButtonVariant =
   | "brushNav"
   | "minimal"
   | "minimalNav"
-  | "buttonStyledAsLink";
+  | "buttonStyledAsLink"
+  | "flat";
```

Comment:
Maybe not to do in this PR, but is "flat" the best description for the button variant? As it's only really used in the curric header it might be worth brainstorming with @HelenHarlow the best style name for this

Reply:
Good point, it could definitely do with being more descriptive